search/includes/classes/class-search.php

```diff
@@ -208,6 +213,13 @@ protected function setup_healthchecks() {
 		add_action( 'init', [ $this->healthcheck, 'init' ] );
 	}
 
+	protected function setup_regular_stat_collection() {
+		$this->field_count_gauge = new FieldCountGaugeJob();
+
+		// Hook into init action to ensure cron-control has already been loaded
+		add_action( 'init', array( $this->field_count_gauge, 'init' ) );
```

Comment:
Since we're not actually registering any jobs any more (piggy backing on the existing one), we can skip this delay and just init right away.